Makefile

```diff
@@ -24,3 +24,28 @@ start_all:
 
 start_all_secure:
 	cd tools && ./start_all_tmux.sh secure
+
+start_test_all:
+	cd tools && chmod +x start_test.sh && ./start_test.sh
+
+clean_all_service:
+	cd lucida \
+	&& cd calendar && make clean \
+	&& cd ../commandcenter && make clean \
+	&& cd ../djinntonic && make clean \
+	&& cd ../imagematching/opencv_imm && make clean \
+	&& cd ../../musicservice && make clean \
+	&& cd ../questionanswering/OpenEphyra && make clean \
+	&& cd ../../weather && make clean \
+	&& cd ../..
+
+all_service:
+	cd lucida \
+	&& cd calendar && make all \
+	&& cd ../commandcenter && make all \
+	&& cd ../djinntonic && make all \
+	&& cd ../imagematching/opencv_imm && make all \
+	&& cd ../../musicservice && make all \
+	&& cd ../questionanswering/OpenEphyra && make all \
+	&& cd ../../weather && make all \
+	&& cd ../..
```

Comment:
same here I think you can just `cd lucida && make all` by listing all directories here it will make it very difficult for users to add their own microservices